.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: go
+sudo: required
+
+go:
+    - 1.9
+
+go_import_path: github.com/clearlinux/mixer-tools
+
+before_install:
+  - sudo apt-get update -qq
+  - go get -u gopkg.in/alecthomas/gometalinter.v1
+  - gometalinter.v1 --install
+
+script:
+   - gometalinter.v1 --deadline=10m --tests --vendor --disable-all --enable=misspell --enable=vet --enable=ineffassign --enable=gofmt --enable=gocyclo --cyclo-over=10 --enable=golint --enable=deadcode --enable=varcheck --enable=structcheck --enable=unused ./...
```

Comment:
Could this line be made into a make target so I can run it locally easier?

Reply:
Yes ^ that was planned but can become part of this patch. I'll update the description.

Reply:
This will be actually added with the CLI update going in as it changes nearly all of the code as is, and addresses the errors generated by this.